lib/charms/nginx_ingress_integrator/v0/ingress.py

```diff
@@ -117,12 +117,16 @@ def _config_dict_errors(self, update_only=False):
         unknown = [
             x
             for x in self.config_dict
-            if x not in REQUIRED_INGRESS_RELATION_FIELDS | OPTIONAL_INGRESS_RELATION_FIELDS
+            if x
```

Comment:
Out of curiosity, what is the reason for this change? Maybe black is configured to a line length of 80?

Reply:
Possibly. The old charm does not have a pyproject.toml file yet, so that might be the case. This will be fixed in future PRs.